frontend/test/metabase/scenarios/dashboard/reproductions/20637-add-series-to-dashcard.cy.spec.js

```diff
@@ -86,7 +86,7 @@ function createQuestionsAndDashboard() {
             ],
           });
 
-          visitDashboard(dashboard_id);
+          cy.visit(`/dashboard/${dashboard_id}`);
```

Comment:
We can always expand the `visitDashboard` helper to account for ALL individual `dashcard` requests, but the code gets complicated really quickly. Without this adjustment, about 99.9% of all tests work just fine.

For cases where we explicitly need to assert on the individual requests, simply do not use the helper.